app/assets/stylesheets/components/app-side.scss

```diff
@@ -0,0 +1,5 @@
+.app-side {
+    @include govuk-responsive-margin(6, "top");
+    padding: govuk-spacing(3);
+    background: govuk-colour("light-grey");
+}
```

Comment:
The indentation looks slightly incorrect here and I see that `stylelint` isn't logging any errors. I think `package.json` might need to be updated like so:
```
  "stylelint": {
    "extends": ["stylelint-config-gds/scss", "stylelint-stylistic/config"]
  },
  "devDependencies": {
    "standardx": "^7.0.0",
    "stylelint": "^15.11.0",
    "stylelint-config-gds": "^1.1.1",
    "stylelint-stylistic": "^0.4.5"
  }
```
which should then log a few indentation Sass errors.

Reply:
Ah - thanks! Have fixed that, and added the lint update as a final commit.

Reply:
Excellent thanks!

As an aside, I wonder if we should include some view formatting. I notice that [Whitehall](https://github.com/alphagov/whitehall/blob/main/Gemfile#L62) use one https://github.com/Shopify/erb-lint which does all the usual linter type things - removing empty lines, standardising quotations etc.

It's fairly easy to setup. You can copy the yml file https://raw.githubusercontent.com/alphagov/whitehall/main/.erb-lint.yml.

Then run like so: ` bundle exec erblint * -a`.

Reply:
Ah, nice. Have added that to the CI in the last commit (and retroactively applied its fix to the various previous commits)